src/homeWork_10/StringMethods.java

```diff
@@ -0,0 +1,41 @@
+package homeWork_10;
+
+public class StringMethods {
+
+    String doсument;
+
+    public static boolean getStart(String doсument) {
+        return doсument.startsWith("555");
+    }
+
+    public static boolean getEnd(String doсument) {
+        return doсument.endsWith("1a2b");
+    }
+
+    public static String getReplace(String doсument) {
+        return doсument.replace("aPs", "***").replace("Odh", "***");
```

Comment:
Document uuid may be different! This method oriented on test data only, it is wrong